### ISSUE TITLE
Navigation breadcrumbs identified by mobile screen readers

### ISSUE THREAD

**Issue by imperator99** [govau/dta-gov-au#283]
# What’s the issue?

On mobile, breadcrumb navigation link are not visible but are read out to screen
reader users and can be tabbed to by keyboard users.

# Why it matters

This has an impact for all mobile screen reader users as navigation that is not visible (or cannot be activated) is read out to users, creating unnecessary noise and providing confusion when navigating through content.

# Our recommendation

Hide content from view as well as from screen reader users and keyboard
